poly-commit/src/streaming_kzg/data_structures.rs

```diff
@@ -140,7 +140,7 @@ where
 
 /// Stream implementation of foleded polynomial.
 #[derive(Clone, Copy)]
-pub struct FoldedPolynomialStream<'a, F, S>(FoldedPolynomialTree<'a, F, S>, usize);
+pub struct FoldedPolynomialStream<'a, F, S>(FoldedPolynomialTree<'a, F, S>);
```

Comment:
cc @mmaker is it okay to delete the field?

Reply:
yes I think so! I think `len` was used at the beginning when I thought we needed to provide shorter length than 2^|challenges| but in the end was not used. I'll fix it if new errors come up.

Reply:
Thanks!